openstack/provision-install-test.sh

```diff
@@ -40,14 +39,14 @@ EOD
 }
 
 # get the options
-while getopts hckLpsS c ; do
+while getopts hcdkLpsS c ; do
```

Comment:
I think the 's' option can safely be removed from here now

Reply:
:+1: